.travis.yml

```diff
@@ -1,4 +1,14 @@
 language: scala
 sudo: false
+
 scala:
-  - 2.11.4
+  - 2.10.4
+  - 2.11.5
+
+jdk:
+  - openjdk7
+  - oraclejdk7
```

Comment:
I'm fine with this, but I'm pretty sure I've seen issues come up only with that JDK on other projects, and since it's currently probably the most widely used of the three, I thought it would be worth testing.

As a side note, I'd like to standardize these points of Travis CI configuration across our OSS Scala projects, and right now we're testing all three for most projects. If we want to drop Oracle's JDK 7 we should probably do it across the board.